python/ray/dashboard/consts.py

```diff
@@ -87,7 +87,7 @@
     METRICS_INPUT_ROOT, "prometheus", "prometheus.yml"
 )
 PARENT_HEALTH_CHECK_BY_PIPE = env_bool(
-    "RAY_enable_pipe_based_agent_to_parent_health_check", False
+    "RAY_enable_pipe_based_agent_to_parent_health_check", True
```

Comment:
is there any reason this wasn't on by default previously? would look at git history to sanity check

Reply:
I found a comment here https://github.com/ray-project/ray/pull/41935#discussion_r1428162075. In fact, the default value on the Python side is not used at all.

Reply:
Make sense. Reverted. Thanks.